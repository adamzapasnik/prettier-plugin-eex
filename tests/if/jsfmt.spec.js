run_spec(__dirname, ['eex']);
