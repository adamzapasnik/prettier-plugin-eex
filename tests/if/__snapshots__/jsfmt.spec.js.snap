// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`js.eex - eex-verify: js.eex 1`] = `
Object {
  Symbol(raw): "<div>
  <%= if ss do %>
    ss
  <% else %>
sss  
  <% end %>
    </div>

    <div>
     <%= if ss do %>
    ss
     <%= if ss do %>
    ss
  <% else %>
sss  
  <% end %>
  <% else %>
   <%= if ss do %>
    ss
  <% else %>
sss  
  <% end %>
sss  
  <% end %>
    </div>


<div
  <%= ddd %>
    class=\\"dd\\"
></div>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div>
  <%= if ss do %>
    ss
  <% else %>
    sss
  <% end %>
</div>

<div>
  <%= if ss do %>
    ss
    <%= if ss do %>
      ss
    <% else %>
      sss
    <% end %>
  <% else %>
    <%= if ss do %>
      ss
    <% else %>
      sss
    <% end %>
    sss
  <% end %>
</div>

<div <%= dd %> class=\\"dd\\"></div>
",
}
`;
